### DISCUSSION TITLE
Add Google SSO for users authentication

### DISCUSSION THREAD

**Discussion by orianec123** [GitbookIO/community#419] [Feature Requests]
We want our customers to have access to the workspace only with authentication. Currently, there is no option for Google authentication (we use Firebase).

**Comment by MadeByPP** [comment]
I hope you vote for ticket #247 . Similar content and more votes. In other words, it can be more noticeable.

**Reply by aikoii** [reply]
👋 @orianec123 , thanks for your suggestion! As @MadeByPP (thank you 🙂) mentioned, I encourage you to join similar discussions or vote 👉 #247. I'll be closing this in favor of that one.
